apps/Wallet/src/types/safe-transaction.ts

```diff
@@ -11,3 +11,21 @@ export type FundUserTransactionBody = {
 export type AddOwnerTransactionBody = {
   ownerAddress: string;
 }
+
+export type AddUserTransactionBody = {
```

Comment:
should these be just one type, UserTransactionBody, or better like this?

Reply:
Maybe good to do cleanup of the owner related code since that wont be necessary anymore with this change

Reply:
using single AddressTransactionBody type